core/internal/settings/settings.go

```diff
@@ -235,6 +240,61 @@ func (s *Settings) GetProgram() string {
 	return s.Proto.Program.GetValue()
 }
 
+// The relative path to the script that created the run, if available.
+func (s *Settings) GetProgramRelpath() string {
+	return s.Proto.ProgramRelpath.GetValue()
+}
+
+// The absolute path to the script that created the run, if available.
+func (s *Settings) GetProgramAbspath() string {
+	return s.Proto.ProgramAbspath.GetValue()
+}
+
+// The arguments passed to the script that created the run, if available.
+func (s *Settings) GetArgs() []string {
+	return s.Proto.XArgs.GetValue()
+}
+
+// The local path to the code directory, if available.
```

Comment:
what's a local path?

Reply:
i wish i knew... it is some launch nonsense... 
but as far as i understand it is the relative path of the code to the cwd.

Reply:
https://github.com/wandb/wandb/blob/main/wandb/sdk/internal/system/system_info.py#L170-L172
found this reference in the artifact context